pkg/api/rpms.go

```diff
@@ -26,6 +26,31 @@ type RepositoryRpmCollectionResponse struct {
 	Links Links            `json:"links"` // Links to other pages of results
 }
 
+type SnapshotErrataListRequest struct {
+	UUID     string   `param:"uuid"`     // Identifier of the repository
+	Search   string   `query:"search"`   // Errata Id to optionally filter-on
+	Type     []string `query:"type"`     // Type to optionally filter-on
+	Severity []string `query:"severity"` // Severity to optionally filter-on
+}
+
+type SnapshotErrata struct {
+	Id              string `json:"id"`
```

Comment:
Can you use ID here?  
```suggestion
	ID              string `json:"id"`
```
We have UUID and IBM Style guide only uses "ID".
if this is already in widespread use then ignore this suggestion.

Reply:
looks like ID and Id have been used interchangeably in [tang](https://github.com/content-services/tang/blob/main/pkg/tangy/rpm.go), i'll try changing this to ID and see what breaks

Reply:
It should be ID, but we haven't been strict about those code style standards.

Reply:
In this case, it's just the name of the struct field. It wouldn't affect the API docs. In the descriptions, it should definitely be ID.

Reply:
got it, changed the descriptions to ID

Reply:
Thank you